src/rdiff_backup/Main.py

```diff
@@ -584,15 +584,18 @@ def backup_warn_if_infinite_regress(rpin, rpout):
         return
 
     relative_rpout_comps = tuple(rpout.path[len(rpin.path) + 1:].split(b'/'))
-    relative_rpout = rpin.new_index(relative_rpout_comps)
-    if not Globals.select_mirror.Select(relative_rpout):
-        return
+    relative_rpout = rpin.new_index(relative_rpout_comps)  # noqa: F841
+    # FIXME: this fails currently because the selection object isn't stored
+    #        but an iterable, the object not being pickable.
+    #        Related to issue #296
+    #if not Globals.select_mirror.Select(relative_rpout):  # noqa: E265
```

Comment:
Tell me if I'm wrong: 
This was checking if the `relative_rpout` is part of the selection. If it was, it doesn't print the error messages.

Reply:
More precisely, it was checking if Select was excluding it from the selection.